### DISCUSSION TITLE
Is there anyway adding config entity_encoding to tinyMCE?

### DISCUSSION THREAD

**Discussion by livlign** [PiranhaCMS/piranha.core#2044] [Q&A] [answered]
I have trouble with the rich text editor when adding content contains special latin characters. These characters are automatically replaced to their escaped in code view mode and when saved to database, for example:

`à  =>  &agrave;
á  =>  &aacute;`

I found out I can bypass this behavior by adding this config to tinyMCE: entity_encoding : "raw"
But the editorconfig.json only support some configs as I can see here, there is no option for entity_encoding
https://github.com/PiranhaCMS/piranha.core/blob/2a58723b51ca408cd1ec5c76d411b36043f8329c/core/Piranha.Manager.TinyMCE/assets/piranha.editor.js

Is there any way I can add this config to tinyMCE? 
For more info, I'm using the piranha.razor template and tweaking with visual code in MacOS

Many Thanks!

**Comment by livlign** [comment]
Okay, so I think I got a workaround. What I've done is:
- cloned the project Piranha.Manager.TinyMCE
- edit the file /assets/piranha.editor.js
- rebuild the project and let my solution use the tweaked dll rather than pulling it from nuget

Apparently this is not a good way, it still working so far.

**Comment by tidyui** [comment, marked as answer]
Hi there! The system with the embedded files works in the way that it checks the local file system first. This means that you should just be able to put a local copy of piranha.editor.js in your application and this will be used instead of the one embedded into the package.

Regards
